docs/articles_en/learn_openvino/openvino_samples/c_sample_hello_classification.rst

```diff
@@ -152,6 +152,4 @@ See Also
 - :doc:`Using OpenVINO™ Samples <openvino_docs_OV_UG_Samples_Overview>`
 - :doc:`Model Downloader <omz_tools_downloader>`
 - :doc:`Convert a Model <openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide>`
-- :doc:`C API Reference <pot_compression_api_README>`
-
-
+- `C API Reference <https://docs.openvino.ai/2023.3/pot_compression_api_README.html>`__
```

Comment:
why does it point to POT?

Reply:
I have updated the link with the https://github.com/openvinotoolkit/openvino/pull/22104/commits/1f4874ea0b0ee61a7924eff4caa71ed6586973de commit